### DISCUSSION TITLE
Mapping a shortcut to custom command

### DISCUSSION THREAD

**Discussion by abudhakeer** [Milkdown/milkdown#409] [Q&A]
I wanted to map a keyboard shortcut to perform a custom action. I found the following code snippet from [here](https://milkdown.dev/using-utils-package). But idk how to integrate this with the editor instance. Also, if i want to perform some custom action which is not related to any editor operation, is this snippet enough or should i need to use the `command` as well?

```
import { $shortcut } from '@milkdown/utils';

const myCommand = $command(/* ... */);

const myShortcut = $shortcut((ctx) => {
    return {
        'Mod-Alt-1': () => myCommand.run(1),
        'Mod-Alt-2': () => myCommand.run(2),
    };
});
```

**Editor instance:**
```

const editor = useEditor((root) =>
    Editor.make()
      .use(nord)
      .use(history)
      .use(slash)
      .use(commonmarkPlugins)
      .use(getCustomShortcuts())
  );
```

**Comment by Saul-Mirone** [comment]
You need to use both `command` and `shortcut`.

**Reply by abudhakeer** [reply]
Thank you for your reply @Saul-Mirone. Can you also help in sharing some examples regd the same? It would be of great help

**Reply by Saul-Mirone** [reply]
Will do in
#288
